pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -91,14 +91,16 @@ def create_account():
 
         accounts = [create_account() for _ in range(num)]
         node.prepare_accounts(accounts,
-                              parent,
-                              balance=0.3,
-                              msg="create passive user")
-        # TODO: this could also be done in parallel, actually in very simple
-        # ways since there are no nonce conflicts (transactions are signed by
-        # different users)
-        for account in accounts:
-            self.register_passive_user(node, account)
+                            parent,
+                            balance=7,
```

Comment:
Do we need this change in balance for this PR?

Reply:
Yes: this PR congests the network, which increases the gas price. I slowly increased balance until I got to 2 there, and at 2 it was still not enough. At that point reproducing the failure took several hours of an overnight run, so I eventually gave up and multiplied the previous `.3` by 20, which AFAIR is our max gas increase factor. This led to a balance of 6, and I added 1 just to be sure.

The result actually did manage to run for a few days in order to generate the 18M users / 19G rocksdb state, so I think it should be enough regardless of how long we let things run :)